lighthouse-core/scripts/roll-to-devtools.sh

```diff
@@ -43,10 +43,10 @@ cp -pPR "$lh_bg_js" "$lh_worker_dir/lighthouse-dt-bundle.js"
 echo -e "\033[96m ✓\033[39m (Potentially stale) lighthouse-dt-bundle copied."
 
 # copy report generator + cached resources into $fe_lh_dir
-cp -r dist/dt-report-resources/ $fe_lh_dir
+cp -r dist/dt-report-resources/* $fe_lh_dir
```

Comment:
I faintly recall commenting that `folder/` vs. `folder/*` vs `folder` seem scary to me.

Is it time to expand the comment? :)

Reply:
I think you were referring to `folder` vs `folder/`. This glob syntax seems obvious to me.

Reply:
I guess my point is, why the change now? Does this have any effect on our situation? If it doesn't, why are we changing it? If it does have an effect and was wrong before, then it probably deserves capturing whatever subtle mistake led to this.

AFAIK this should be the same if `$fe_lh_dir` is always going to exist, is that right?

Reply:
It was always wrong and I didn't notice during any of the rolls to Chromium.

This landed 4 months ago, but for each roll actually just ignored / deleted the `dt-report-assets` b/c the Chromium changes that necessitated it (saving as HTML) were still pending review. That just changed during this latest roll. So I've now noticed.

https://github.com/GoogleChrome/lighthouse/pull/7567#discussion_r266978276 was wrong (see https://askubuntu.com/a/1151667/399299)

I'll try `cp -r dist/dt-report-resources/. $fe_lh_dir` (seems better?) and update with a comment.

Reply:
Hm that's weird maybe it's OS/shell/cp-impl dependent?

On my machine with zsh, `cp -r folder/ other` and `cp -r folder/ other/` both result in the files of `folder` being copied into `other` without the extra `folder/` layer suggested by  https://askubuntu.com/a/1151667/399299

Reply:
Either way, very clearly stating the desired effect of the cp command in a comment with the important variations that *won't* work seems like a great idea to me :)

Reply:
i think my issue was that the folder already existed, perhaps from much earlier messing around, so in that case it does copy into that folder.

Reply:
so it would work in most cases, but not in the one small case where you happen to have a `dt-report-resources` folder :)

Reply:
oh `dt-report-resources` already existed *in* `other/`? interesting, I guess `cp` be all kinds of 😑